README.md

```diff
@@ -220,11 +220,22 @@ expect({'foo': 'bar'}).to.exclude('baz')
 
 #### throw
 
-Asserts that the target throws an exception
+Asserts that the target throws an exception (or its parent)
```

Comment:
I don't think these two sentences have the same meaning.
For example, we have function `foo()` that raises an `ArithmeticError`. We also know that `DivisionByZero` is a subclass of `ArithmeticError`.
Now consider the following codes.
```python
expect(lambda o: foo()).to.throw(ArithmeticError)
```
This passed.
```python
expect(lambda o: foo()).to.throw(ZeroDivisionError)
```
This failed.